app/code/core/Mage/Bundle/Model/Product/Type.php

```diff
@@ -1009,8 +1009,8 @@ public function processBuyRequest($product, $buyRequest)
         $option     = $buyRequest->getBundleOption();
         $optionQty  = $buyRequest->getBundleOptionQty();
 
-        $option     = (is_array($option)) ? array_filter($option, 'intval') : array();
-        $optionQty  = (is_array($optionQty)) ? array_filter($optionQty, 'intval') : array();
+        $option     = (is_array($option)) ? array_filter($option, '\intval') : array();
```

Comment:
Any reason you don't use short array here and other places?
```suggestion
        $option     = (is_array($option)) ? array_filter($option, '\intval') : []);
```

Reply:
No reason - "A few code style changes have been added by mistake". Just kept it, b/c it doesnt hurt.